samples/chat/README.md

```diff
@@ -37,10 +36,10 @@ Please refer to the root level [README](../../README.md#requirements) for prereq
         GetChatState: [GET] http://localhost:7071/api/chats/{chatId}
 
         PostUserResponse: [POST] http://localhost:7071/api/chats/{chatId}
-
-        OpenAI::ChatBotEntity: entityTrigger
     ```
 
+    Note for running the post user response function provided in the sample, please specify a model name in the environment variable `AZURE_DEPLOYMENT_NAME`. This value can be an Azure deployment name or a GPT model name.
```

Comment:
How do users know what value to provide for `AZURE_DEPLOYMENT_NAME`? Can we provide an example? Also, if this is required for the sample, then we should add it with a default value in the local.settings.json file.

Reply:
Updated the documentation! Let me know if this makes sense